pytest-selenium/requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ certifi==2023.7.22
 cffi==1.15.0
 cfgv==3.3.1
 chardet==4.0.0
-chromedriver-binary==119.0.6045.105
+chromedriver-binary==107.0.5304.62
```

Comment:
Did you mean to downgrade the Chrome binary?

Reply:
it was not part of this PR, so I made it match whatever is in main.
I change it locally while running slelnium tests and it got committed